src/llmq/quorums_commitment.h

```diff
@@ -94,8 +96,13 @@ class CFinalCommitment
         obj.push_back(Pair("llmqType", (int)llmqType));
         obj.push_back(Pair("quorumHash", quorumHash.ToString()));
         obj.push_back(Pair("signersCount", CountSigners()));
+        obj.push_back(Pair("signers", CLLMQUtils::ToHexStr(signers)));
         obj.push_back(Pair("validMembersCount", CountValidMembers()));
+        obj.push_back(Pair("validMembers", CLLMQUtils::ToHexStr(validMembers)));
```

Comment:
It's probably better to either use "01010001110001" (binary ascii) here.

Reply:
> It's probably better to use "01010001110001" (binary ascii) here.

I only partly agree here so far.. please tell me, to get your reasoning, why do you think it's better?

Reply:
For a quorum of 400 members that's going to be a rather long binary string... I think hex is fine - it's much more compact (which I guess is preferable for rpc output) and can be converted to any format you like afterwards.

Ping @cofresi and @shumkov: guys, what kind of format do you expect for these fields?

Reply:
I think hex should be fine. For us I don't think it makes a difference, so I would go with the more compact format. What do you think @shumkov and @antouhou?